content/spin/v2/http-trigger.md

```diff
@@ -267,31 +267,14 @@ For a full Rust SDK reference, see the [Rust Spin SDK documentation](https://doc
 
 > [**Want to go straight to the reference documentation?**  Find it here.](https://fermyon.github.io/spin-js-sdk/)
 
-In JavaScript or TypeScript, the handler is identified by name.  It must be called `handleRequest`.  The way you declare it is slightly different between the two languages.
+In TypeScript, the user must a define a function named `handler` which the SDK attaches to the [`fetch` event listener](https://developer.mozilla.org/en-US/docs/Web/API/FetchEvent).
```

Comment:
A wasi-http incoming event is translated in the engine to be a fetchEvent ([ref](https://developer.mozilla.org/en-US/docs/Web/API/FetchEvent))

Reply:
Maybe writing it as "the `FetchEvent` listener" rather than "the `fetch` event" would be clearer, but again I don't know what is idiomatic in JS.  For me it would still invite the question "which FetchEvent listener" (because I know it as the HTTP trigger) but that may be another idiom thing!